api/auth_utils.py

```diff
@@ -0,0 +1,32 @@
+import hashlib
+import secrets
+import string
+
+
+HASH_ALGORITHM = 'sha256'
+NUM_HASH_ITERATIONS = 100000
+HASH_OUTPUT_LEN = 32
+
+SALT_NUM_BYTES = 32
+
+MIN_REQUIRED_PASSWORD_LEN = 7
+
+

```

Comment:
no need for all these newlines. just need one between the last import and before the function